[EDU-4101] - add whoami command docs (#823)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx

---
title: Azion CLI whoami
description: Azion CLI whoami command displays the email of the user currently logged in.
meta_tags: cli, next, javascript, astro, vue, react, edge computing
namespace: documentation_cli_whoami
menu_namespace: cliMenuAlpha
permalink: /documentation/devtools/cli/whoami/
---

> 
> ## whoami
> 
> Shows the email of the currently logged in user on Azion CLI.
> 

Reviewer: ```suggestion

Shows the email of the currently logged in user on Azion CLI.

---

```

---
title: Azion CLI whoami
description: Azion CLI whoami command displays the email of the user currently logged in.
meta_tags: cli, next, javascript, astro, vue, react, edge computing
namespace: documentation_cli_whoami
menu_namespace: cliMenuAlpha
permalink: /documentation/devtools/cli/whoami/
---

> Shows the email of the currently logged in user on Azion CLI.

Reviewer 2: Displays the email of the currently logged-in user on Azion CLI.

:::note 
In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
:::

---

## Usage

```bash
$ azion whoami
```

## Local options

### help

> The `-h` or `--help` flag displays more information about the `azion whoami` command. It is an optional flag.

Reviewer 3: ```suggestion
The `-h` or `--help` flag displays more information about the `azion whoami` command. It's an optional flag.
```
